csrankings.csv

```diff
@@ -10034,6 +10035,7 @@ Kenneth J. Supowit,Ohio State University,https://cse.osu.edu/people/supowit.1,NO
 Kenneth L. Calvert,University of Kentucky,https://www.cs.uky.edu/~calvert,-6v-CdkAAAAJ
 Kenneth M. Anderson,University of Colorado Boulder,http://www.cs.colorado.edu/~kena,wOO8rEMAAAAJ
 Kenneth M. Hopkinson,Air Force Institute of Technology,https://www.afit.edu/BIOS/bio.cfm?facID=87,NOSCHOLARPAGE
+Kenneth O. Stanley,University of Central Florida,https://www.cs.ucf.edu/~kstanley/,6Q6oO1MAAAAJ

```

Comment:
My understanding is that he is no longer full-time at UCF but instead at OpenAI.